lib/storage.js

```diff
@@ -104,28 +110,27 @@ async function getSource(paramsSource, inDirectory, disableSourceDownload) {
     return getAsset(paramsSource, inDirectory, name, disableSourceDownload);
 }
 
-async function putRendition(rendition) {
+async function putRendition(rendition, directories) {
     // Note: validation has happened in validate.js before
 
     if (process.env.WORKER_TEST_MODE) {
-        if(rendition.name.includes('post-')) {
-            // to verify post-processing renditions, since their names contain `post-` in the beginning
-            // and asset-compute-cli command run-worker only looks at the file named `rendition*.*`
-            rendition.name = rendition.name.substring(5);
-            
-            const newPath = path.join(rendition.directory, rendition.name);
-            await fs.rename( rendition.path, newPath);
-            
-            rendition.path = newPath;
+        if (rendition.directory === directories.post) {
```

Comment:
The name `directories.post` is confusing when used in a storage library that potentially could do HTTP(S) POST requests (see one of the previous comments). Suggesting to rename it from `post` -> `postprocessing`.

Reply:
Discussed above.